content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -155,33 +162,41 @@ int CalculateTestSquare( const int number)
 
 TEST(MathTests, FastSquare_Integer_Squared1)
 {
+    // despite the name suggesting this verifies the math of computing a square
+    // this test trusts the function is correct, and only proves an identity property
     int tenSquared = Math::FastSquare(10);
     int result = Math::FastSquare(10);
     ASSERT_EQ(result, tenSquared);
 }
 
 TEST(MathTests, FastSquare_Integer_Squared2)
 {
+    // relies on a duplicate implementation of the production code which contains
+    // the same bug, and only helps prove the production bug was not changed
```

Comment:
```suggestion
    // Relies on a duplicate implementation of the production code that contains
    // the same bug, and proves the production bug was not changed
```

Reply:
Will accept all these changes except the omission of "only"